src/components/Filters/index.tsx

```diff
@@ -362,23 +363,29 @@ class Filters extends React.Component<FiltersProps, FiltersState> {
 
 	public render() {
 		const { filters } = this.state;
+		const hookValues = this.props.compact || [false];
 
 		return (
 			<FilterWrapper mb={3}>
 				<Flex justifyContent="space-between">
 					{this.shouldRenderComponent('add') && (
-						<Button
-							mr={30}
-							disabled={this.props.disabled}
-							primary
-							icon={<FontAwesomeIcon icon={faFilter} />}
-							onClick={() =>
-								this.setState({ showModal: true, editingFilter: null })
-							}
-							{...this.props.addFilterButtonProps}
-						>
-							Add filter
-						</Button>
+						<UseBreakpointWrapper values={hookValues}>
```

Comment:
See my comment above, let's do that for this component. I realize that this one is much bigger, so option 2. might be better. The fact that this component is pretty big means it needs to be refactored anyway, so lets do it in 2 commits, 1 that refactors it, and the second one that adds the compact prop